openedx/features/marketplace/models.py

```diff
@@ -1,6 +1,30 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+from openedx.custom.db.fields import MultiSelectWithOtherField
+from openedx.features.idea.models import VisualAttachment, Location, OrganizationBase
+from openedx.features.marketplace.constants import (ORGANIZATION_SECTOR_CHOICES, ORGANIZATIONAL_PROBLEM_CHOICES,
+                                                    USER_SERVICES)
+
+
+class Challenge(OrganizationBase, Location, VisualAttachment):
```

Comment:
Have we missed the textfield shown in this screenshot?
![image](https://user-images.githubusercontent.com/28558328/80098454-58ca8280-8586-11ea-8cd9-e40fecc45c41.png)